packages/wholly-sheet/package.json

```diff
@@ -37,6 +37,7 @@
     "uuid": "^8.3.1"
   },
   "devDependencies": {
+    "@looker/sdk-node": "^21.0.4",
```

Comment:
This one needed to be fixed as well. Needed by a test that still worked (I guess through the magic of lerna) but I felt the eslint error needed to be fixed sooner rather than later